.github/workflows/ci.yaml

```diff
@@ -13,14 +13,14 @@ on:
 
 env:
   GO_VERSION: '1.20'
-  TINYGO_VERSION: 0.30.0
+  TINYGO_VERSION: 0.31.2
   # Run e2e tests against latest two releases and latest dev
   ENVOY_IMAGES: >
+    envoyproxy/envoy:v1.30-latest
```

Comment:
maybe we want to do in a previous step?

Reply:
I opened https://github.com/corazawaf/coraza-proxy-wasm/pull/284

Reply:
Merged.